### DISCUSSION TITLE
Mac OS Client 2.2.0 odd behavior

### DISCUSSION THREAD

**Discussion by BTGUK** [kaiserdj/Darkorbit-client#24] [General]
Hey there!

I just tried the 2.2.0 client for mac OS (great work btw!). Game-wise the client works fine, however it exhibits a strange behavior where after launch the app icon in the dock and the program's menu bar disappears and only the DO window is left. The expected behavior would be that the app icon remains in the dock until the user closes the program (i.e. the client). 

Also, the mac OS app icon looks weirdly out of place, I would suggest putting a white square with rounded corners behind it so it looks more like other Mac OS icons.

**Comment by ivan-kirov** [comment]
I have noticed the same things. Also the browser plugin for master password can't be closed, even if we click close nothing happens so it seems to be stuck there.

**Comment by kaiserdj** [comment]
I recommend you create an issue to be able to better work with the problems or errors. And please, could you show with images or a video what you would like to change or what you have problems with?
